### ISSUE TITLE
How to read the stored data?

### ISSUE THREAD

**Issue by centi** [ameltech/sme-nt3h1x01-library#3]
Hi,

I'm making my first steps with SmartEverything NFC but have some troubles.

I managed to run your examples for writing successfully (was able to read the data with mobile phone application). Next I wanted to write some data from the mobile phone and then read it on my SE device. But there are no "read" versions for the storeText/storeURI/... methods.

How do I read the stored data? Could you point me to some example or documentation on how to do it?

My desired usecase is to use some external device (mobile phone) to send data to the SE NFC and let the SE device read the message and react. Is this even possible?

**Comment by searobin** [comment]
Hi Ivan.

At the moment the NFC lib does not support write operations.

If you would like to provide support to this open project,
you can add the required API to the related git repository.
We can add them to the next release of the lib.

Adding Mik for any additional information.

Thanks

Severino

**Comment by bkharrab** [comment]
I don't think that you can read data with the NT3H1201, it's just an tag with an I2C interface.
source : [http://www.nxp.com/products/identification-and-security/nfc-and-reader-ics/connected-tag-solutions/ntag-ic-energy-harvesting-nfc-forum-type-2-tag-with-field-detection-pin-and-ic-interface:NT3H1201W0FHK#featuresExpand](url)
